### ISSUE TITLE
Delete from Stream in Output cluster(System/Outputs) showing 404 error and restart graylog service require to forget disconnection of output

### ISSUE THREAD

**Issue by makstock** [Graylog2/graylog2-server#8714]
## Expected Behavior
Remove output from the stream via the Global i.e. "System -> Outputs -> Delete from stream" should be possible. After removing output it should delete/remove all connection from Graylog cluster nodes without restarting Graylog service.

## Current Behavior
1) The main problem is that when you disconnect/delete the output from a stream, it does not get disconnected from all cluster members but only from that one node that you are accessing the Web UI from. You have to restart the Graylog services on the other nodes to make them forget about the disconnected output.

2) Try to remove output from the stream via the global "System -> Outputs -> Delete from stream" it will fail with a 404
<img width="1438" alt="Screenshot 2020-08-06 at 19 28 32" src="https://user-images.githubusercontent.com/62546449/89540758-1a965d80-d81b-11ea-98e7-75c20a691b87.png">

## Possible Solution
1) After delete or disconnect output from stream, the restart of Graylog service require to remove all output connection from cluster nodes.

2) There is a workaround to delete output from Stream--> Manage Outputs -->Delete from Stream 
<img width="1437" alt="Screenshot 2020-08-06 at 19 25 39" src="https://user-images.githubusercontent.com/62546449/89540483-b83d5d00-d81a-11ea-92a8-b71e7736dbac.png">

## Steps to Reproduce (for bugs)
1. run a TCP sink on a Linux box with netcat (# nc -l -p 4102 -k)
2. select a stream, manage output, create GELF TCP output (default options)
3. notice the node you are actually accessing the UI from
4. remove the output from the stream (channel disconnected message will only appear on the node where you access the UI from)
5. stop the TCP sink/netcat on the Linux box
6. see the other nodes in the cluster complaining about output destination refusing connection

## Context
https://support.graylog.com/#ticket/zoom/3762

## Your Environment

* Graylog Version: 3.3.3
* Java Version: NA
* Elasticsearch Version:
* MongoDB Version: 
* Operating System: 
* Browser version:

**Comment by thll** [comment]
This issue is about two problems:
1.) Removing an output from a stream doesn't have the same effect on all nodes
2.) The button to remove an output from all streams is broken.

### Fix for issue 1.)
While working on https://github.com/Graylog2/graylog2-server/issues/5634, I've noticed that when deleting an output, we disassociate it from all streams on all nodes, but keep the output running on all nodes except the one that received the API request in the first place.
I'll try to extend the implementation of #12195 to provide a fix for this particular case.

### Fix for issue 2.)
We should either remove the button or reword it to `Remove from all streams` and implement a new API endpoint for that.
